jekyll/_cci2/server/v4.3/installation/phase-1-prerequisites.adoc

```diff
@@ -555,6 +555,32 @@ aws iam create-role --role-name circleci-s3 --assume-role-policy-document file:/
   ]
 }
 ----
++
+NOTE: If you wish to store artifacts which are larger than 5GB, you will need to enable non-presigned URLs. To do this you will need your IRSA role to assume itself. Replace your trust policy above with the contents below.
++
+[source, json]
+----
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "Federated": "<OIDC_PROVIDER_ARN>"
+      },
+      "Action": [
+        "sts:AssumeRoleWithWebIdentity",
+        "sts:AssumeRole"
+      ],
+      "Condition": {
+        "StringEquals": {
+          "<OIDC_PROVIDER_URL>:sub": "system:serviceaccount:<K8S_NAMESPACE>:object-storage"
+        }
+      }
+    }
+  ]
+}
```

Comment:
I don't think this is the correct Trust Policy for non-presigned Role. It looks like it just adds the `sts:AssumeRole` `Action` to the existing statement but it should be in an additional statement:
```json
{
  "Effect": "Allow",
  "Principal": {
    "AWS": "<ROLE_ARN>"
  },
  "Action": "sts:AssumeRole"
}
```
So the whole thing would be:
```json
{
  "Version": "2012-10-17",
  "Statement": [
    {
      "Effect": "Allow",
      "Principal": {
        "Federated": "<OIDC_PROVIDER_ARN>"
      },
      "Action": "sts:AssumeRoleWithWebIdentity",
      "Condition": {
        "StringEquals": {
          "<OIDC_PROVIDER_URL>:sub": "system:serviceaccount:<K8S_NAMESPACE>:object-storage"
        }
      }
    },
    {
      "Effect": "Allow",
      "Principal": {
        "AWS": "<ROLE_ARN>"
      },
      "Action": "sts:AssumeRole"
    }
  ]
}
```